### ISSUE TITLE
License

### ISSUE THREAD

**Issue by andre-campos** [cbrueffer/pep8-git-hook#5]
Not really an issue, more of a question, really.  No license really means all rights reserved. In that case, I'd need authorization from you to run or modify this. Do you grant such authorization?

PS: You might want to add a license to this in order to prevent questions like this ;)

**Comment by cbrueffer** [comment]
Good point; the reason I haven't added a license is that this is a fork from a gist (see README) which doesn't have a license itself. I can't license the code without permission of the original author.
